acceptance/openstack/rds/v3/rds_test.go

```diff
@@ -2,46 +2,35 @@ package v3
 
 import (
 	"testing"
+	"time"
 
 	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/openstack"
+	networking "github.com/opentelekomcloud/gophertelekomcloud/acceptance/openstack/networking/v1"
 	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/tools"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/common/pointerto"
 	"github.com/opentelekomcloud/gophertelekomcloud/openstack/common/tags"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/rds/v3/backups"
 	"github.com/opentelekomcloud/gophertelekomcloud/openstack/rds/v3/instances"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/rds/v3/logs"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/rds/v3/security"
 	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
 )
 
-func TestRdsList(t *testing.T) {
-	client, err := clients.NewRdsV3()
-	th.AssertNoErr(t, err)
-
-	listOpts := instances.ListRdsInstanceOpts{}
-	allRdsPages, err := instances.List(client, listOpts).AllPages()
-	th.AssertNoErr(t, err)
-
-	rdsInstances, err := instances.ExtractRdsInstances(allRdsPages)
-	th.AssertNoErr(t, err)
-
-	for _, rds := range rdsInstances.Instances {
-		tools.PrintResource(t, rds)
-	}
-}
-
 func TestRdsLifecycle(t *testing.T) {
 	client, err := clients.NewRdsV3()
```

Comment:
skip this test from running in zuul, too long to complete

Reply:
if os.Getenv("RUN_RDS_LIFECYCLE") == "" {
		t.Skip("too slow to run in zuul")
	}

Reply:
done